schema/mysql/schema.sql

```diff
@@ -404,7 +406,9 @@ CREATE TABLE servicegroup (
 
   PRIMARY KEY (id),
 
-  INDEX idx_servicegroup_name (name) COMMENT 'Host/service/service group list filtered by service group name'
+  INDEX idx_servicegroup_display_name (display_name) COMMENT 'Servicegroup list filtered/ordered by display_name',
+  INDEX idx_servicegroup_name_ci (name_ci) COMMENT 'Servicegroup list filtered using quick search',
+  INDEX idx_servicegroup_name (name) COMMENT 'Host/service/service group list filtered by service group name; Servicegroup detail filter'
```

Comment:
Host list? By service group name? How is this even possible?

Reply:
Like this:
![image](https://github.com/Icinga/icingadb/assets/16668527/b383d09d-a08d-4eec-ac72-079aaa45d0e3)